modules/zone/zone.go

```diff
@@ -93,82 +56,86 @@ func (s *Lookup) DoZonefileLookup(record *dns.Token) (interface{}, zdns.Status,
 	lookup, _ := s.Factory.Subfactory.MakeLookup()
 	var nameserver string
 	var domain string
-	var notify chan bool
+	var waiting []*dns.Token
 	prefix := s.Factory.Factory.GlobalConf.NamePrefix
```

Comment:
Same question as above. This is only used once in the function inside of an if statement. Why is it being defined here?